airmozilla/popcorn/tests/test_views.py

```diff
@@ -0,0 +1,51 @@
+from airmozilla.base.tests.testbase import DjangoTestCase, Response
+from airmozilla.main.models import Event
+
+import mock
+import json
+
+from nose.tools import eq_, ok_
+
+from funfactory.urlresolvers import reverse
+
+
+class TestPopcornEvent(DjangoTestCase):
+    fixtures = ['airmozilla/manage/tests/main_testdata.json']
+    main_image = 'airmozilla/manage/tests/firefox.png'
+    title = 'Test event'
+    location = 'http://localhost'
```

Comment:
Since it's only ever used in one place, move these two things into the test. 
Leave the `main_image` since its value actually matters.